x-pack/plugins/upgrade_assistant/__jest__/client_integration/helpers/app_context.mock.ts

```diff
@@ -48,7 +48,7 @@ const shareMock = sharePluginMock.createSetupContract();
 shareMock.url.locators.get = (id: IdKey) => ({
   useUrl: (): string | undefined => idToUrlMap[id],
   getUrl: (params: Record<string, any>): string | undefined =>
-    `${idToUrlMap[id]}?${new URLSearchParams(convertObjectValuesToString(params)).toString()}`,
+    `${idToUrlMap[id]}?${new URLSearchParams(stringifySearchParams(params)).toString()}`,
```

Comment:
I think this name makes more sense if the function returns a string. In https://github.com/elastic/kibana/pull/111252#discussion_r703693648 I suggested moving `new URLSearchParams().toString()` into the function to accomplish that -- WDYT?

Reply:
Somehow I miss read the previous comment, i patched that up and will merge when ci finishes 🚀